packages/frontend/src/components/CookiePopUp.tsx

```diff
@@ -27,6 +29,9 @@ const CookiePopUp : React.FC = () => {
       <CookieConsent
         location="none"
         buttonText="Accept"
+        onAccept={() => {
+          track(EVENT_NAME.COOKIE_ACCEPTED)
+        }}
```

Comment:
This is where we can track the initial  cookie rejection by calling onReject function here, but i couldnt add it cause we havent initialized amplitude... which is what i wanted to discuss with you